spec/models/stats_spec.rb

```diff
@@ -6,8 +6,17 @@
   subject { described_class.new }
 
   describe "#new_reg" do
-    it "returns the correct value" do
-      expect(subject.new_reg).to eq(25)
+    it "only counts registrations in the correct date range" do
```

Comment:
I've just got a couple of suggestions to make based on what we say in our [ruby-services-team tests guidance](https://github.com/DEFRA/ruby-services-team/blob/master/tests.md#tests).

You'll have seen we're not as strict as our guidance tells us to be when using [traits](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#traits). However in this case I reckon there is an argument for having a trait called something like `:was_renewed` rather than setting the property when we create the registration.

The second is our notes round [Assemble-Act-Assert](https://github.com/DEFRA/ruby-services-team/blob/master/tests.md#assemble---assert---act). Basically we could do with a new line between the `create()` calls and the `expect()` in these tests.